docs/bulkIngest.md

```diff
@@ -24,9 +24,9 @@ This tutorial uses the following Java classes.
  * [BulkIngestExample.java] - ingest the data using map reduce
  * [VerifyIngest.java] - checks that the data was ingested
  
-Remember to copy the accumulo-examples-\*.jar to Accumulo's 'lib/ext' directory.
+Remember to copy the accumulo-examples\*.jar to Accumulo's 'lib/ext' directory.
```

Comment:
is the backslash before the * needed here?

Reply:
Yes, because it is markdown.